reviewrot/basereview.py

```diff
@@ -82,6 +87,64 @@ def check_request_state(self, created_at,
                 raise ValueError("Invalid duration type: %s" % duration)
         return True
 
+    def _decode_response(self, response):
+        """
+        Remove Gerrit's prefix and convert to JSON.
+        Returns:
+            Converted JSON content
+        Raises:
+            ValueError if the content is not in proper json format.
+        """
+        gerrit_json_prefix = ")]}\'\n"
+        content = response.content.strip()
+        try:
+            if response.encoding:
+                content = content.decode(response.encoding)
+            response.raise_for_status()
+            content_type = response.headers.get('content-type', '')
+            if content_type.split(';')[0] != 'application/json':
```

Comment:
If we must inspect the content type header of the response, it would be more error-proof to check everything and not just the first portion. Ex:
```python
if 'application/json' in content_type.split(';'):
```